.github/workflows/ds.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os:
+        arch:
```

Comment:
Huch?

Reply:
fixed

Reply:
Sorry, could you please change both to be "os"? Not mission-critical but more consistent with other workflows and GitHub actions documentation.

Reply:
Discussed in https://github.com/openmls/openmls/pull/1090#discussion_r1027878324.